app/controllers/admin/ministries_controller.rb

```diff
@@ -19,6 +19,29 @@ def show
 	end
 
 	def update
+		@location_id = params[:ministry][:location_id]
+		@ministry_level = params[:ministry][:level]
+
+		if @location_id.blank?
+			params[:ministry][:location_id] = 0
+		else
+			@location_type = Location.find(@location_id).location_type
+			if @ministry_level == "national"
```

Comment:
use switch case.

Reply:
Converted "if-else" statements in 'create' and 'update' methods to 'switch-case' statements.